### ISSUE TITLE
Install on Windows

### ISSUE THREAD

**Issue by edsu** [DocNow/twarc#194]
This is a screenshot of a failure to install on Windows, since v1.3.1. I think the issue is related to the fix for #145.

![image001](https://user-images.githubusercontent.com/33829/33578163-8bd062fc-d913-11e7-9ccd-9163983f8576.png)

**Comment by edsu** [comment]
I just uploaded v1.3.2 to PyPI with this commit: ece7bea63944f895fa418e97f8c9d1977302eba7 -- I'm checking to see if this resolves the issue.

**Comment by yasarhussain** [comment]
I just tested on my machine and it's not working.  It's looking for twarc.py in the root folder, not the bin folder.

**Comment by edsu** [comment]
@yasarhussain the latest screencap seems to indicate otherwise?

![error v1 3 2_take two](https://user-images.githubusercontent.com/33829/33579361-932af356-d917-11e7-9455-bc4840b014cb.PNG)

**Comment by yasarhussain** [comment]
Yeah that worked, sorry I was pulling in the cached 1.3.1 version.

**Comment by edsu** [comment]
@yasarhussain no worries -- thanks for giving it a try!

**Comment by edsu** [comment]
@yasarhussain no worries -- thanks for giving it a try!

**Comment by edsu** [comment]
Adjusting the manifest seems to have convinced setuptools to include bin/twarc.py in the distribution tarball.

**Comment by gwiedeman** [comment]
On Windows, 1.3.4 now looks like Scripts\twarc.py is importing itself when you `import twarc` which has no `main()`

I get `AttributeError: module 'twarc' has no attribute 'main'`

printing `twarc.__file__` after import shows `C:\python36\Scripts\twarc.py` and if I rename the twarc.py file it works. This was the case with both Python 3.6.1 and 3.6.4.

In 0.8.x I had always just made a powershell alias to the twarc.py to avoid the previous issue. Not sure how to set it up the correct way.

**Comment by alblaine** [comment]
@gwiedeman What did you rename the twarc.py file?

**Comment by gwiedeman** [comment]
@alblaine You can rename it anything and use that as the command. It you make it `twarc2.py` it will run with `twarc2 search whatever` and it will stop trying to import itself.

I see some docs about setting up a library to run as a script with an -m switch, but I'm not sure if there is a reason twarc isn't set up that way.

**Comment by edsu** [comment]
Is it possible that you have twarc installed at another location and it is finding it there?

```
% python
Python 3.6.0 (default, Dec 29 2016, 02:55:04)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import twarc
>>> print(twarc.__file__)
/home/ed/.pyenv/versions/3.6.0/lib/python3.6/site-packages/twarc/__init__.py
```

**Comment by gwiedeman** [comment]
Possible, as I had 0.8.x installed before, but it doesn't look like it:

        PS C:\scripts> python
        Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)] on win32
        Type "help", "copyright", "credits" or "license" for more information.
        >>> import twarc
        >>> print(twarc.__file__)
        C:\python36\lib\site-packages\twarc\__init__.py

This is the full trace for running "twarc":

       PS C:\scripts> twarc
       Traceback (most recent call last):
         File "C:\python36\Scripts\twarc.py", line 3, in <module>
           import twarc
         File "C:\python36\Scripts\twarc.py", line 4, in <module>
           twarc.main()
       AttributeError: module 'twarc' has no attribute 'main'


Not sure if its something to do with my local setup, but it definitely looks like its importing itself. I reproduced it on another Windows machine with Python 3.4 as well. If I add this main class at the start of `C:\python36\Scripts\twarc.py` it prints `test` twice

       #!c:\python36\python.exe
       
       def main():
              print ("test")

       import twarc
       twarc.main()

**Comment by edsu** [comment]
Sorry for the delay in addressing this. I believe that this should be fixed in v1.3.5. Can you upgrade and see if that's the case?

    pip install --upgrade twarc

**Comment by edsu** [comment]
Thanks for reporting it, and for testing!
